### PR TITLE
Reduce usage of pytest test-parameters

### PR DESCRIPTION
**Pull request by ydirson**
There is no real need to store the parameters coming from the CLI in a (single) test param, and it makes it much harder to describe test dependencies.

This PR deals with `--hosts` and `--sr-disk` (easily tested), applies same strategy to  `--second-network` (untested), and removes unused `--additional-repos`.

The same can be applied at least to `--sr-device-config`, but I'm leaving it out for now as it is more complex to test.

### CODE REVIEW

**Review by stormi — COMMENTED** [review]
This looks good to me. I suggest you handle the remaining parameters in the same PR, on which we will anyway run the full testsuite to make sure nothing regressed.

**Review by benjamreis — APPROVED** [review]

File: conftest.py
Comment (Nambrok) [inline conftest.py:235]: Should be `sr_disk_4k`. It's a different parameter than normal `sr_disk`.

**Review by Nambrok — CHANGES_REQUESTED** [review]

File: conftest.py
Comment (ydirson) [reply conftest.py]: right, fixed

**Review by Nambrok — APPROVED** [review]

**Comment by stormi** [comment]
LGTM, provided a full suite run doesn't reveal any drawbacks.

**Comment by ydirson** [comment]
Added a fix for a wrong usage of a package-scoped fixture (and checked no other such wrong usage exists in the codebase)

**Comment by ydirson** [comment]
Rebased onto master after #232 merge

**Pull request merged**
